tests/covalent_dispatcher_tests/_core/execution_test.py

```diff
@@ -294,50 +294,58 @@ def workflow(x):
     assert result_object._error == "The following tasks failed:\n0: failing_task"
 
 
+@pytest.mark.asyncio
 async def test_run_workflow_does_not_deserialize(mocker):
     """Check that dispatcher does not deserialize user data when using
     out-of-process `workflow_executor`"""
 
-    from dask.distributed import LocalCluster
-
-    from covalent._workflow.lattice import Lattice
-    from covalent.executor import DaskExecutor
-
-    lc = LocalCluster()
-    dask_exec = DaskExecutor(lc.scheduler_address)
-
-    @ct.electron(executor=dask_exec)
+    @ct.electron(executor="local")
     def task(x):
         return x
 
-    @ct.lattice(executor=dask_exec, workflow_executor=dask_exec)
+    @ct.lattice(executor="local", workflow_executor="local")
     def workflow(x):
         # Exercise both sublatticing and postprocessing
-        sublattice_task = ct.lattice(task, workflow_executor=dask_exec)
-        res1 = ct.electron(sublattice_task(x), executor=dask_exec)
+        sublattice_task = ct.lattice(task, workflow_executor="local")
+        res1 = ct.electron(sublattice_task(x), executor="local")
         return res1
 
+    # asyncio.set_event_loop(event_loop)
```

Comment:
Can you remove this line as well 😄 .

Reply:
Thanks, I must have forgotten about this 😅